zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/InsertTrace.java

```diff
@@ -51,7 +53,7 @@
       this.session = session;
       this.timestampCodec = new TimestampCodec(session);
       Insert insertQuery =
-          QueryBuilder.insertInto("traces")
+          QueryBuilder.insertInto(TRACES)
```

Comment:
I noticed we never used references for the main table! This allows reference search when you are trying to figure out impact of a table. Using text is less precise as the word "traces" is too common